sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -122,7 +122,16 @@ def __init__(self, engine: trt.ICudaEngine):
     self.outputs = []
     self.gpu_allocations = []
     self.cpu_allocations = []
-    """Setup I/O bindings."""
+
+    # TODO(https://github.com/NVIDIA/TensorRT/issues/2557):
```

Comment:
Should we have an issue on our end to track this?

Reply:
sure, opened: https://github.com/apache/beam/issues/25387